Lombiq.Tests.UI/Extensions/ImageSharpImageExtensions.cs

```diff
@@ -29,7 +29,23 @@ public static Image CalcDiffImage(this Image actual, Image expected)
         return ImageSharpCompare.CalcDiffMaskImage(actualStream, expectedStream);
     }
 
-    private static Stream ToStream(this Image image)
+    /// <summary>
+    /// Clones the <see cref="Image"/>.
+    /// </summary>
+    /// <param name="image">The source <see cref="Image"/> instance.</param>
+    /// <returns>Cloned <see cref="Image"/> instance.</returns>
+    public static Image Clone(this Image image)
```

Comment:
How about using `Image.CloneAs()`?

Reply:
I don't want to play with pixel formats.

Reply:
Isn't one already used by default, or configured implicitly, when doing anything with `Image`?

Reply:
There is a new `Clone` extension in the new version of `SixLabors.ImageSharp` package, I'll use that.